service/systemd/service.go

```diff
@@ -230,8 +237,18 @@ func (s *Service) readConf() (common.Conf, error) {
 	var conf common.Conf
 
 	data, err := Cmdline{}.conf(s.Service.Name, s.DirName)
-	if err != nil {
+	errString := fmt.Sprintf("%v", err)
```

Comment:
You can get the error message by calling `err.Error()` - it's the only method on the error interface.